app/src/ui/add-repository/licenses.ts

```diff
@@ -87,9 +95,9 @@ export async function writeLicense(
     'email',
     'project',
     'description',
-    'year',
+    'year'
```

Comment:
I think our prettier config expects this trailing `,` and will error otherwise

Reply:
:+1: